src/routes/middleware/validators/recipe.validator.ts

```diff
@@ -1,13 +1,50 @@
-import { body, ValidationChain } from "express-validator";
+import {
+  body,
+  param,
+  ValidationChain,
+  CustomValidator,
+  query,
+} from "express-validator";
 
-const title = body("title").notEmpty();
-const description = body("description").notEmpty();
-const url = body("url").notEmpty();
-const id = body("id").notEmpty();
+const title = body("title")
+  .notEmpty()
+  .isString()
+  .isLength({ min: 16, max: 80 });
```

Comment:
We don't need to set a min of 16 characters for title right? I think we can require at least ~3 characters and this should be enough